tools/repo/notify.py

```diff
@@ -132,6 +136,21 @@ def slo_max(self):
     async def stalled_prs(self):
         return (await self.tracked_prs)["stalled_prs"]
 
+    @async_property(cache=True)
```

Comment:
small nit - i think this only gets called once so no need to cache it - but pretty minor point